File: output/ChatGPT/avatar/Python/Java/atcoder_ABC123_B.java
import java.util.ArrayList;
import java.util.Collections;
import java.util.Comparator;
import java.util.List;
import java.util.Scanner;
import java.lang.Math;

public class atcoder_ABC123_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        List<int[]> menu = new ArrayList<>();
        
        for (int i = 0; i < 5; i++) {
            String m = scanner.nextLine();
            int e = Integer.parseInt(m.substring(m.length() - 1));
            if (e == 0) {
                e = 10;
            }
            menu.add(new int[]{Integer.parseInt(m), e});
        }
        
        Collections.sort(menu, new Comparator<int[]>() {
            public int compare(int[] a, int[] b) {
                return Integer.compare(a[1], b[1]);
            }
        });
        
        int ans = 0;
        for (int i = 1; i < menu.size(); i++) {
            ans += Math.ceil(menu.get(i)[0] / 10.0) * 10;
        }
        
        System.out.println(ans + menu.get(0)[0]);
    }
}

File: output/ChatGPT/avatar/Python/C++/atcoder_ABC123_B.cpp
#include <iostream>
#include <algorithm>
#include <cmath>
using namespace std;

int main() {
    int menu[5];
    for (int i = 0; i < 5; i++) {
        string m;
        cin >> m;
        int e = m.back() - '0';
        if (e == 0) {
            e = 10;
        }
        menu[i] = stoi(m);
    }
    sort(menu, menu + 5, [](int a, int b) { return a % 10 < b % 10; });
    int ans = 0;
    for (int i = 1; i < 5; i++) {
        ans += ceil(menu[i] / 10.0) * 10;
    }
    cout << ans + menu[0] << endl;

    return 0;
}